### DISCUSSION TITLE
using `resources:` field in `_quarto.yml`

### DISCUSSION THREAD

**Discussion by ijlyttle** [quarto-dev/quarto-cli#3763] [Q&A]
I am working on a PR for pins (rstudio/pins-r#692), and have gotten myself confused about how to use `resources:` in `_quarto.yml`. I don't know if I have found a bug, or if I am just doing it wrong. Hopefully someone can point me in the right direction.

I see this is discussed also in https://github.com/quarto-dev/quarto-cli/discussions/3443.

Here's my `_quarto.yml` file:

```yml
project:
  type: website
  resources:
    - data/board_one

resources:
  - data/board_two
```

`board_one` and `board_two` are directories; my goal is that they be copied recursively into `_site/`.

I find this works for `board_two`, even though this `resources:` isn't in the [documented location](https://quarto.org/docs/websites/website-tools.html#site-resources).

However, for `board_one`, it copies the file in the folder, but not the folder inside. I get these warnings when I render locally:

```
WARNING: File 'data/board_one/mtcars/20221226T175420Z-c2702' was not found.
WARNING: File 'data/board_one/mtcars/20221226T175420Z-c2702/data.txt' was not found.
WARNING: File 'data/board_one/mtcars/20221226T175420Z-c2702/mtcars.json' was not found.
```

I have a (hopefully) reproducible [repo](https://github.com/ijlyttle/quarto-pins), rendered and published with an action; here's the [`data` directory](https://github.com/ijlyttle/quarto-pins/tree/gh-pages/data). 

I am using quarto 1.3.56.

Thanks!

**Comment by cscheid** [comment]
Before I try to repro this more thoroughly, can you check if adding a trailing backslash to your path helps?

**Reply by ijlyttle** [reply]
Just added trailing backlashes to both resources: no change in either resource, for rendering both locally and using actions.

Thanks for having a look!

**Reply by cscheid** [reply]
I'll create an issue and follow up when I'm back home.

**Reply by ijlyttle** [reply]
Thanks! Of course, I'm at your service to help.
